tests/test_network.py

```diff
@@ -129,30 +129,38 @@ def test_network_interfaces(shell):
 
 
 @pytest.mark.lg_feature("ptx-flavor")
-def test_network_nfs_io(shell):
+def test_network_nfs_io(env, target, shell):
     """Test nfs share io"""
-    ptx_works = shell.target.env.config.get_target_option(shell.target.name, "ptx-works-available")
-    assert len(ptx_works) > 0
-
-    mount = shell.run_check("mount")
-    mount = "\n".join(mount)
+    ptx_works = set(env.config.get_target_option(target.name, "ptx-works-available"))
 
+    readable = set()
+    writeable = set()
+    missing = set()
     # Iterate over all available shares and check whether io operation is possible
     for ptx_work in ptx_works:
-        assert ptx_work in mount
-
-        dir_contents = shell.run_check(f"ls -1 {ptx_work}")
-        # make sure the directories contain something
-        assert len(dir_contents) > 0
-
-        shell.run_check(f"cd {ptx_work}")
+        # Check if an automount unit has been created
+        unit_name = f'{ptx_work[1:].replace("/", "-")}.automount'
```

Comment:
Although the paths follow a fixed pattern, maybe it's better to use systemd-escape for this?

Reply:
Replaced this by checking if we have an `autofs` mounted. That's even more straight forward.